components/ResultsPageComponents/SharedComponents/ContentStyles.jsx

```diff
@@ -0,0 +1,17 @@
+import { Box } from "@chakra-ui/react";
+
+const ResultContentSection = (props) => {
```

Comment:
```suggestion
export default function ResultContentSection(props) {
```

Reply:
I was going to have more than one component defined in the file. If you prefer one component per file, I am happy to change it to suit.